### PR TITLE
[Core][Geometries] Interface geometries test

### PR DESCRIPTION
**Pull request by ipouplana**
As discussed in PR #3808. I just created three simple tests for the interface geometries. They can't be made in the same way as the standard geometries because of the special features of interfaces (they can have volume = zero, and the determinant of the Jacobian may not be determined).

I also exported them to python, but maybe it's not necessary...

### CODE REVIEW

File: kratos/utilities/geometry_tester.h [hunk 1]
```diff
@@ -388,6 +394,31 @@ class GeometryTesterUtility
...
+
+        if(std::abs(geom.Area() - expected_area) > 1e-14)
+        {
+            error_msg << "Geometry Type = " << "Kratos_QuadrilateralInterface3D4" << " --> " 
+                      << " error: area returned by the function geom.Area() does not deliver the correct result " << std::endl;
+            succesful=false;
+        }
+
+        if(succesful == false)
+            KRATOS_THROW_ERROR(std::logic_error,"", error_msg.str());
```
Comment (philbucher) [inline kratos/utilities/geometry_tester.h]: `KRATOS_ERROR_IF_NOT(succesful) << error_msg.str() << std::endl;`

File: kratos/utilities/geometry_tester.h [hunk 2]
```diff
@@ -532,6 +563,32 @@ class GeometryTesterUtility
...
+
+        if(std::abs(geom.Volume() - expected_vol) > 1e-14)
+        {
+            error_msg << "Geometry Type = " << "Kratos_HexahedraInterface3D8" << " --> " 
+                      << " error: volume returned by the function geom.Volume() does not deliver the correct result " << std::endl;
+            succesful=false;
+        }
+
+        if(succesful == false)
+            KRATOS_THROW_ERROR(std::logic_error,"", error_msg.str());
```
Comment (philbucher) [inline kratos/utilities/geometry_tester.h]: same

File: kratos/utilities/geometry_tester.h [hunk 3]
```diff
@@ -609,6 +666,32 @@ class GeometryTesterUtility
...
+
+        if(std::abs(geom.Volume() - expected_vol) > 1e-14)
+        {
+            error_msg << "Geometry Type = " << "Kratos_PrismInterface3D6" << " --> " 
+                      << " error: volume returned by the function geom.Volume() does not deliver the correct result " << std::endl;
+            succesful=false;
+        }
+
+        if(succesful == false)
+            KRATOS_THROW_ERROR(std::logic_error,"", error_msg.str());
```
Comment (philbucher) [inline kratos/utilities/geometry_tester.h]: same

**Review by philbucher — COMMENTED** [review]
some minor comments otherwise this looks fine to me

Thx for the fast response!

@loumalouomega knows the geometry tests better than me, maybe he can also take a look?

**Review by philbucher — APPROVED** [review]
Thanks!

**Comment by ipouplana** [comment]
I'd also like to know what @loumalouomega thinks about it. Especially whether they need to be exported to python or not...

**Comment by loumalouomega** [comment]
I am OK with this, you can merge, we can discuss further details in the
future.

**Pull request merged**